pkg/webhook/risingwave_validating_webhook.go

```diff
@@ -132,6 +132,21 @@ func (v *RisingWaveValidatingWebhook) validateGlobal(path *field.Path, global *r
 		}
 	}
 
+	// Validate labels of the RisingWave's service type
+	if global.ServiceType != "" && global.ServiceType != corev1.ServiceTypeClusterIP && global.ServiceType != corev1.ServiceTypeNodePort &&
```

Comment:
The validation is already done by the CRD generated from markers. You can see it here:
https://github.com/risingwavelabs/risingwave-operator/blob/2af5c44854dc522356d358d9742c98170e68636e/apis/risingwave/v1alpha1/risingwave_types.go#L544